mkdocs.yml

```diff
@@ -137,6 +137,16 @@ nav:
       - Get facts from CloudVision: docs/how-to/v3/cv_facts_v3.md
     - Development tips & tricks: docs/installation/collection-installation.md
     - Activate logging: docs/how-to/debug.md
+  - Schema documentation:
+    - CV Modules v3:
+      # - Module cv_configlet_v3: docs/schema/cv_tag_v3.md
+      - Module cv_container_v3: docs/schema/cv_container_v3.md
```

Comment:
I would suggest removing the "module" prefixes here, it's already stated in the upper link